930-delegate-tasks/src/main/java/io/harness/delegate/task/servicenow/ServiceNowError.java

```
@@ -0,0 +1,29 @@
+package io.harness.delegate.task.servicenow;
+
+import static io.harness.annotations.dev.HarnessTeam.CDC;
+
+import io.harness.annotations.dev.OwnedBy;
+import io.harness.jackson.JsonNodeUtils;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.experimental.FieldDefaults;
+import lombok.extern.slf4j.Slf4j;
+
+@OwnedBy(CDC)
+@Data
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Slf4j
+public class ServiceNowError {
+  String errorMessage;
+  public ServiceNowError(JsonNode node) {
+    String message = JsonNodeUtils.getString(node, "message");
+    String detail = JsonNodeUtils.getString(node, "detail");
```

if both are blank then we can throw an exception and hence default to raw exception

---

Done